modules/op-creating-default-repository-cr.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assemblies:
+// * pac/using-repository-crd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-default-repository-cr_{context}"]
+= Creating the default Repository custom resource
```

Comment:
In this doc https://pipelinesascode.com/docs/install/global_repositories_setting/ its called as Global Repository 

cc @chmouel should we keep Global instead of Defaults :thinking:

Reply:
yes global is how it's called, we really need to make sure to don't have different names between upstream and downstream